cmake/BuildUnicyclePlanner.cmake

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2018  iCub Facility, Istituto Italiano di Tecnologia
+# Authors: Stefano Dafarra <stefano.dafarra@iit.it>
+# CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
+
+include(YCMEPHelper)
+include(FindOrBuildPackage)
+
+find_or_build_package(iDynTree 0.9.2 QUIET)
```

Comment:
If this depends on iDynTree `devel`, I think we should do a release of iDynTree before merging this PR.

Reply:
Yes. It needs ``optimalcontrol`` stuff.